lib/utils.ts

```diff
@@ -395,6 +407,47 @@ async function requestAsync(
 		opts.headers = nativeHeaders;
 	}
 
+	const { FormData: NodeFormData } =
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
+		require('formdata-node') as typeof import('formdata-node');
+	const { FormDataEncoder } =
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
+		require('form-data-encoder') as typeof import('form-data-encoder');
+
+	const nodeForm = new NodeFormData();
+	const form = IS_BROWSER ? new FormData() : nodeForm;
```

Comment:
How about making these lines a `getForm()` helper, that depending on `IS_BROWSER` imports only the correct one, so that the CLI doesn't have to load both on startup (hence less files to parse)?

Reply:
@thgreasi I will refactor it, but the `require('formdata-node')` will happen still. What we can minimize is the `require('form-data-encoder')`  by moving it into the "else" condition in case it has a blob.